lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -150,20 +152,24 @@ export abstract class BaseManagedIdentitySource {
                 networkRequest.computeParametersBodyString();
         }
 
+        const networkClient = managedIdentityRequest.disableInternalRetries
```

Comment:
Isn't there an easier way to inject the `disableInternalRetries` in this class ? Similar to how you inject networkClient?